lib/ssl/doc/src/standards_compliance.xml

```diff
@@ -1492,8 +1492,8 @@
       <row>
         <cell align="left" valign="middle"></cell>
         <cell align="left" valign="middle">status_request (RFC6066)</cell>
-	<cell align="left" valign="middle"><em>NC</em></cell>
-	<cell align="left" valign="middle"></cell>
+	<cell align="left" valign="middle"><em>C</em></cell>
```

Comment:
this is wrong. nothing is implemented on server side within this PR.
Certificate with status request extension should be marked as compliant in client section above and not here.

Reply:
609f4ac122

Reply:
You can mark it as PC (partially complient)

Reply:
4.4.2.1. OCSP Status and SCT Extensions -> mark as PC

Reply:
c7424afdb4